packages/venia-concept/src/components/Checkout/wrapper.js

```diff
@@ -4,7 +4,7 @@ import { array, bool, func, object, oneOf, shape, string } from 'prop-types';
 
 import { Util } from '@magento/peregrine';
 import { getShippingMethods } from 'src/actions/cart';
-import {
+import actions, {
```

Comment:
Looks like you're only using `actions.reset`, let's just add `reset` to the list of named imports.

Reply:
Also, it doesn't look like `src/actions/checkout` exports a function named `reset` ?